src/main/java/frc/robot/commands/turret/TurretAimAtTargetCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.turret;
+
+import frc.controls.AxisCode;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** Aims the turret at the target */
+public class TurretAimAtTargetCommand extends CommandBase {
+
+  public TurretAimAtTargetCommand() {
+    addRequirements(Robot.TURRET_SWIVEL_SUBSYSTEM);
+  }
+
+ 
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    Robot.TURRET_SWIVEL_SUBSYSTEM.setShot(Constants.TURRET_DEFAULT_PID);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    double angle = Robot.GAMEPAD.getAxis(AxisCode.RIGHTSTICKX);
```

Comment:
This will eventually conflict with driving, this same stick controls robot rotation. Can we move this to the operator controller for now?

Reply:
This was for testing w/o limelight but now will be commented out